lib/libutee/include/tee_api_types.h

```diff
@@ -57,12 +59,20 @@ typedef union {
 	} value;
 } TEE_Param;
 
+typedef TEE_Param REE_Param;
+
 /*
  * The type of opaque handles on TA Session. These handles are returned by
  * the function TEE_OpenTASession.
  */
 typedef struct __TEE_TASessionHandle *TEE_TASessionHandle;
 
+/*
+ * The type of opaque handles on TA Session. These handles are returned by
+ * the function TEE_OpenREESession.
+ */
+typedef struct __TEE_REESessionHandle *TEE_REESessionHandle;
```

Comment:
This should go into `lib/libutee/include/tee_api_types_extensions.h`

Reply:
Since, `UUID` typedef is suggested to be removed. Do you think, it should be removed as well then?

Reply:
I don't think so. A TEE session handle is not the same as a REE session handle and both could have the same values without pointing to the same thing.
OTOH, a UUID is a UUID, and the type `TEE_UUID` is one way of encoding it and can be used everywhere in the TEE-related code.

Reply:
Thanks for confirming on UUID. 
For the session handle, while recalling from memory, I almost forgot my own code ;).